README.md

```diff
@@ -120,7 +117,7 @@ Native provider doesn't run any extra layer/process at the moment.
 *For this example we will use the `macos` version of the executable*
 
 ```bash
-./zombienet-macos
+❯ ./zombienet-macos
```

Comment:
to make all bash snippets alike

applicable to all changes around the project's docs